services/rook-ceph-cluster/1.10.1/defaults/cm.yaml

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
insights has a lot of critical Polaris findings that we should address if possible

Reply:
created https://d2iq.atlassian.net/browse/D2IQ-93353 to resolve this